ls1intum/Artemis
# `Development`: Move node memory leak workaround

### Checklist
#### General

### Motivation and Context
It seems there have been already multiple workarounds for memory leaks like [in this PR](https://github.com/ls1intum/Artemis/pull/4094).
These workarounds are so far just used for the GitHub Actions (build and some tests) and explained in the docs. 
Therefore, newcomers to the project would have to set `--max-old-space-size` themselves and for each other new build or test via GitHub actions or other build agents you would have to set an environment variable as it has been so far.

### Description
The solution would be to either fix all of the builds by introducing a `.npmrc` with the workaround parameter, or by appending it to each build command for node in `package.json`.
Also, the manual for this parameter has been added and the style of the parameter was matched to the official docs.

### Steps for Testing

- Build the client or both locally and verify GitHub actions are still building as supposed to

#### Code Review
#### Manual Tests
